### PR TITLE
add start typescript setup

### PR DESCRIPTION
**Pull request by denar90**
- typescript configurations
- sheets.js -> sheets.ts

### CODE REVIEW

**Comment by pedro93** [comment]
Ping @paulirish, any objection to this?

File: types/types.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  type: string;
+  options: {
+    spreadsheetId: string;
+    tableName: string;
+    clientSecret: Object;
+  };
+}
+
+interface MetricsResults {
+  timestamps: Array<any>;
```
Comment (samccone) [inline types/types.ts]: would prefer if we defined the type instead of any

Comment (denar90) [inline types/types.ts]: Good point. I was thinking to do that after merging https://github.com/paulirish/pwmetrics/pull/50

**Comment by denar90** [comment]
@samccone thanks for the review 👍  I'm merging this one, if you have any comments for improving this stuff I can make it in another PR.

**Comment by paulirish** [comment]
Lgtm!

**Pull request merged**